dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/ResourceCandidate.h

```diff
@@ -23,7 +23,7 @@ struct ResourceCandidate : ResourceCandidateT<ResourceCandidate>
     void SetQualifierValuesFromContext(Microsoft::Windows::ApplicationModel::Resources::ResourceContext context);
 
 private:
-    hstring m_stringData;
+    hstring m_stringData{};
```

Comment:
hstring is a c++ type that has its own initializion, no need to init here.

Reply:
Hmmm I saw that there was a C26495 on it so thought I had to